lib/govuk_navigation_helpers/taxonomy_sidebar.rb

```diff
@@ -19,8 +22,20 @@ def taxons
           title: parent_taxon.title,
           url: parent_taxon.base_path,
           description: parent_taxon.description,
+          related_content: content_related_to(parent_taxon),
         }
       end
     end
+
+    def content_related_to(taxon)
```

Comment:
This call cannot fail, could you please make sure we rescue potential issues here?